### PR TITLE
Change VSTS to Azure DevOps

### PR DESCRIPTION
**Pull request by mhdbouk**
Since VSTS is now Azure DevOps, I've update all the markdown that contain `VSTS` and replace it with `Azure DevOps`

### CODE REVIEW

File: Engineering/SourceControl.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Source Control
-There are many different options when working with Source Control. In [CSE](../CSE.md) we use [VSTS](https://csesd.visualstudio.com/_projects) for private repositories and [GitHub](https://github.com/) for public repositories.
+There are many different options when working with Source Control. In [CSE](../CSE.md) we use [Azure DevOps](https://csesd.visualstudio.com/_projects) for private repositories and [GitHub](https://github.com/) for public repositories.
```
Comment (hxlnt) [inline Engineering/SourceControl.md]: - Consider lowercasing "Source Control."
- Consider removing the link for Azure DevOps, as this project should not be externally accessible. Or, change to something like https://azure.microsoft.com/en-us/services/devops/, which is a landing page for describing the features of Azure DevOps.

Comment (mhdbouk) [reply Engineering/SourceControl.md]: Hello Rachel, updated the pull request. Thank you for your feedback

File: Engineering/SourceControlDetails.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ Let's use the following naming conventions for branches:
  * release branches: `release/release_name`
 
 ## Restrict access to default branches
-* In VSTS it's recommended to specify the **minimum number of required reviewers** as well as the **commit strategy** if you want to enforce [squash merging](https://docs.microsoft.com/en-us/vsts/git/merging-with-squash?view=vsts). Here some details on [VSTS branch policies](https://docs.microsoft.com/en-us/vsts/git/branch-policies?view=vsts)
+* In Azure DevOps it's recommended to specify the **minimum number of required reviewers** as well as the **commit strategy** if you want to enforce [squash merging](https://docs.microsoft.com/en-us/vsts/git/merging-with-squash?view=vsts). Here some details on [Azure DevOps branch policies](https://docs.microsoft.com/en-us/vsts/git/branch-policies?view=vsts)
```
Comment (zmmille2) [inline Engineering/SourceControlDetails.md]: nit: should update link to `https://docs.microsoft.com/en-us/azure/devops/repos/git/branch-policies?view=azure-devops`

Comment (mhdbouk) [reply Engineering/SourceControlDetails.md]: Link updated

**Comment by mhdbouk** [comment]
What is keeping this pull request from being approved.. ?

**Comment by mhdbouk** [comment]
@zmmille2, @nmiodice can you approve this pull request to close it?

**Comment by fnocera** [comment]
The playbook has been given a facelift and some of the structure has changed - we are also working on making the PR process quicker, please reach out to me with any questions. I will close this PR as will be hard to merge back in given the changes that have happened.

**Pull request closed without merging**